java/arcs/core/crdt/CrdtSet.kt

```diff
@@ -262,6 +262,37 @@ class CrdtSet<T : Referencable>(
             override fun toString(): String = "CrdtSet.Operation.Remove($clock, $actor, $removed)"
         }
 
+        /**
+         * Represents the removal of all items from a [CrdtSet]. If an empty [clock] is given,
+         * all items in the set are removed; otherwise, only those items dominated by the clock
+         * are removed. This allow actors with write-only access to a model to operate without
+         * needing to synchronise the clock data.
+         */
+        open class Clear<T : Referencable>(
+            val actor: Actor,
+            override val clock: VersionMap = VersionMap()
+        ) : Operation<T>() {
+            override fun applyTo(data: Data<T>, isDryRun: Boolean): Boolean {
```

Comment:
would be good to have some unit tests for this

Reply:
Argh, I updated HandleManagerTestBase.kt to cover the new approach, but forgot to put some in CrdtSetTest. Will update.

Reply:
Done.